src/components/look-controls.js

```diff
@@ -211,6 +232,17 @@ module.exports.Component = registerComponent('look-controls', {
   onMouseDown: function (event) {
     this.mouseDown = true;
     this.previousMouseEvent = event;
+
+    var sceneEl = this.el.sceneEl;
+    var canvasEl = sceneEl.canvas;
+
+    if (!this.pointerLocked) {
```

Comment:
Yeah, I've had problems with picking up pointerlockerror and using that detect that the user doesn't want to lock their mouse. I can see that in a situation where you have multiple a-frame components, it might be nicer to use drag to look instead of grabbing pointer lock. Maybe we could make it a parameter on look-controls where pointer lock can be enabled or disabled?